pkg/plugins/golang/options.go

```diff
@@ -91,7 +98,7 @@ func (opts Options) UpdateResource(res *resource.Resource, c config.Config) {
 		res.Controller = true
 	}
 
-	if opts.DoDefaulting || opts.DoValidation || opts.DoConversion {
+	if opts.DoDefaulting || opts.DoValidation || opts.DoConversion || opts.Spoke != "" {
```

Comment:
Have intentionally left the `DoConversion` flag's implementation as it is to preserve the current functionality. Hence, if someone just specifies the `--conversion` flag in the command, we will generate the default message which we do now.